how-to-docs/drone-how-to.md

```diff
@@ -0,0 +1,691 @@
+# Drone CI Guide
+
+- Setup
+  - [Install Drone CLI](#install-drone-cli)
+  - [Activate your pipeline](#activate-your-pipeline)
+- Adding a repository to Drone
+  - [Configure your pipeline](#configure-your-pipeline)
+  - [Signature](#signature)
+- Publishing Docker images to
+  - [Quay](#publishing-to-quay)
+  - [Artifactory](#publishing-to-artifactory)
+- Deployments
+  - [Deployments and promotions](#deployments-and-promotions)
+  - [Deploying to DSP](#deploying-to-dsp)
+  - [Versioned deployments](#versioned-deployments)
+  - [Ephemeral deployments](#ephemeral-deployments)
+- Pull requests
+  - [PR Builder](#drone-as-a-pull-request-builder)
+- [QAs](#qas)
+- [Snippets](drone-snippets.md)
+
+## Install Drone CLI
+
+- [Github drone instance](https://drone.digital.homeoffice.gov.uk): https://drone.digital.homeoffice.gov.uk
+- [Gitlab drone instance](https://drone-gitlab.digital.homeoffice.gov.uk/): https://drone-gitlab.digital.homeoffice.gov.uk/
+
+Download and install the [Drone CLI](http://readme.drone.io/0.5/reference/cli/overview/) from the official website.
```

Comment:
The URL here needs fixing (see [my PR](https://github.com/UKHomeOffice/central-devteam-board/pull/165/files) in the other repo).